lib/Sema/CSSimplify.cpp

```diff
@@ -3768,14 +3768,19 @@ ConstraintSystem::SolutionKind ConstraintSystem::simplifyMemberConstraint(
       // Check if we have unviable candidates whose reason for rejection
       // is either UR_InstanceMemberOnType or UR_TypeMemberOnInstance
 
+      SmallVector<std::pair<OverloadChoice, MemberLookupResult::UnviableReason>,
+                  4>
+          filteredURs;
+      llvm::copy_if(
```

Comment:
`filteredURs` and `choices` could be a single collection there is no need to copy data back and forth.

Reply:
@xedin Done!